### PR TITLE
Chevron now 16px in modern theme

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-16px-chevron](https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-16px-chevron)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-16px-chevron](https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-16px-chevron)

**Review by Blackbaud-ToddRoberts — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-16px-chevron](https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-16px-chevron)

**Review by Blackbaud-ToddRoberts — DISMISSED** [review]

File: src/app/public/modules/chevron/chevron.component.html [hunk 1]
```diff
@@ -9,14 +9,32 @@
...
+      aria-hidden="true"
+      class="sky-chevron-part sky-chevron-right"
+    >
+    </i>
+  </ng-container>
+  <ng-container *skyThemeIf="'modern'">
+    <span
+      class="chevron-glyph-container"
+    >
+      <i
```
Comment (Blackbaud-TrevorBurch) [inline src/app/public/modules/chevron/chevron.component.html:28]: Not a huge deal - but we could technically pull the inner portion here out into a template that we then transclude  into both  containers. I don’t know that it is worth holding this up but would ensure that any future changes we make would only need to be made one place in the HTML.

**Review by Blackbaud-TrevorBurch — APPROVED** [review]
Left one comment - but I don’t think it is worth holding this up for.

File: src/app/public/modules/chevron/chevron.component.html (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/chevron/chevron.component.html]: Good suggestion. Done!

**Review by Blackbaud-TrevorBurch — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-16px-chevron](https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-16px-chevron)

**Pull request merged**